docker-compose.yaml

```diff
@@ -125,7 +125,7 @@ services:
     image: bandchain_flusher:latest
     networks:
       bandchain:
-    command: sh -c "sleep 30 && python main.py init bandchain test --db postgres:postgrespassword@172.18.0.88:5432/postgres"
+    command: sh -c "sleep 30 && python main.py init bandchain test --db postgres:postgrespassword@172.18.0.88:5432/postgres && python main.py hasura postgrespassword --db postgres:postgrespassword@172.18.0.88:5432/postgres"
```

Comment:
If you want docker to use new user, please use the new user on hasura service too.